package.json

```diff
@@ -120,6 +121,9 @@
         "webpack-dev-server": "^2.11.3",
         "webpack-merge": "^3.0.0"
     },
+    "resolutions": {
```

Comment:
`Gulp` blows up with the latest node because this package doesn't support it. This ensures that we use a version that works.